custom-stylelint-plugins/no-missing-parenthesis.js

```diff
@@ -0,0 +1,22 @@
+const stylelint = require('stylelint');
+
+const ruleName = "custom-rule/no-missing-parenthesis";
```

Comment:
I'm not sure this is needed - does the [custom-property-no-missing-var-function](https://stylelint.io/user-guide/rules/custom-property-no-missing-var-function) rule and the general error capturing not identify this?

Reply:
This rule only reports custom properties that are defined within the same source.
It will not throw error for this issue:
.spectrum-Well {
  background-color: (--spectrum-well-background-color);
  border-color: var(--spectrum-well-border-color);
}

It will only throw error here:
  :root { --foo: red; }
    a { color: --foo; }